src/nifreeze/data/dmri.py

```diff
@@ -342,3 +366,87 @@ def load(
         dwi_obj.brainmask = np.asanyarray(mask_img.dataobj, dtype=bool)
 
     return dwi_obj
+
+
+def find_shelling_scheme(
```

Comment:
This felt like a dwi-specific manipulation routine, I moved it here for that reason.